packages/web/src/common/store/notifications/fetchNotifications.ts

```diff
@@ -16,50 +20,20 @@ export function* fetchNotifications(config: FetchNotificationsParams) {
     timeOffset = Math.round(new Date().getTime() / 1000), // current unix timestamp (sec)
     groupIdOffset
   } = config
-  const getFeatureEnabled = yield* getContext('getFeatureEnabled')
-  const remoteConfig = yield* getContext('remoteConfigInstance')
-
-  yield* call(remoteConfig.waitForRemoteConfig)
-
-  const useDiscoveryNotifications = yield* call(
-    getFeatureEnabled,
-    FeatureFlags.DISCOVERY_NOTIFICATIONS
-  )
-
-  const discoveryNotificationsGenesisUnixTimestamp = remoteConfig.getRemoteVar(
-    IntKeys.DISCOVERY_NOTIFICATIONS_GENESIS_UNIX_TIMESTAMP
-  )
 
-  const shouldFetchNotificationFromDiscovery =
-    useDiscoveryNotifications &&
-    discoveryNotificationsGenesisUnixTimestamp &&
-    timeOffset > discoveryNotificationsGenesisUnixTimestamp
-
-  if (shouldFetchNotificationFromDiscovery) {
-    return yield* call(fetchDiscoveryNotifications, {
-      limit,
-      timeOffset,
-      groupIdOffset
-    })
-  }
-
-  return yield* call(fetchIdentityNotifications, limit, timeOffset)
-}
-
-function* fetchIdentityNotifications(limit: number, timeOffset: number) {
-  const audiusBackendInstance = yield* getContext('audiusBackendInstance')
-
-  return yield* call(audiusBackendInstance.getNotifications, {
+  return yield* call(fetchDiscoveryNotifications, {
     limit,
-    timeOffset
+    timeOffset,
+    groupIdOffset
   })
 }
 
 function* fetchDiscoveryNotifications(params: FetchNotificationsParams) {
```

Comment:
Maybe we consolidate `fetchDiscoveryNotifications` into `fetchNotifications`? Don't think the wrapper fn is necessary anymore

Reply:
yeah I'll do this in a followup!